.github/workflows/test.yml

```diff
@@ -19,9 +19,10 @@ jobs:
           go-version: 1.18.x
           check-latest: true
       - name: Check out source code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           ref: ${{ env.PR_COMMIT_SHA }}
+          submodules: true
```

Comment:
Unfortunately, this is the checkout of the downstream test, not the target repo for testing. That change needs to happen here:
https://github.com/pulumi/action-test-provider-downstream/blob/7a4e3ebe0a1260750cb0d17afb28083f99e7b049/src/main.ts#L98